### PR TITLE
chore: change listSunnyProjects endpoint to serve only approved projects

### PR DESCRIPTION
**Pull request by hussedev**

### CODE REVIEW

File: src/server/api/routers/projects.ts [hunk 1]
```diff
@@ -249,30 +244,44 @@ export const projectsRouter = createTRPCRouter({
...
+              where: {
+                AND: [
+                  createDataFilter("type", "bytes32", "application"),
+                  createDataFilter("round", "bytes32", round.id),
+                  { id: { in: approvedProjectIds } },
+                ],
+              },
+            },
+            ["id", "decodedDataJson", "recipient"],
+          );
```
Comment (carlbarrdahl) [inline src/server/api/routers/projects.ts:274]: Nice performance improvement of only fetching approval attestations first.

File: src/server/api/routers/projects.ts [hunk 2]
```diff
@@ -305,8 +312,8 @@ export const projectsRouter = createTRPCRouter({
               ...project,
               metadata,
               metrics,
-              status,
-              nextPage: cursor + 1
+              status: "approved",
```
Comment (carlbarrdahl) [inline src/server/api/routers/projects.ts:315]: This might not actually be needed anymore?

**Review by carlbarrdahl — APPROVED** [review]
Nicely done!

File: src/server/api/routers/projects.ts [hunk 3]
```diff
@@ -249,30 +244,44 @@ export const projectsRouter = createTRPCRouter({
...
-            });
+          const attestationFetcher = createAttestationFetcher({ round });
 
-          const statusByProjectId: Record<string, ApplicationStatus> =
-            await getApplicationsStatusMapById({
-              round,
-              projectAttestations,
-            });
+          // Fetch application approvals for round by admins
+          const approvals = await attestationFetcher(["approval"], {
```
Comment (carlbarrdahl) [inline src/server/api/routers/projects.ts:250]: Nice performance improvement of only fetching approval attestations first.

Comment (hussedev) [reply src/server/api/routers/projects.ts:250]: thanks for suggesting it!

File: src/server/api/routers/projects.ts (hunk 2 above)
Comment (hussedev) [reply src/server/api/routers/projects.ts]: done

**Pull request merged**